doc/ARCHITECTURE.md

```diff
@@ -1,29 +1,55 @@
 
+### CI-CD
+The changes made in project's code base adheres to a continuous integration and delivery (CI/CD) pipeline. This guarantees that any alterations undergo rigorous testing and validation before being released to the production environment.
+
+## Technical decision
 In this section you should list all technical things you use like frameworks, tools, languages and provide explanations of why you have choosen them. Think about e.g. what are the main reasons you choose a framework, what are the advatages you need for your project? Try to be as clear as possible so also new joiners understand why your setup is the best for your project.
 
-## APIs
+### Web components
+Our design system includes a wide range of reusable UI components, each carefully designed and implemented using the [Lit](https://lit.dev/docs/). These web components cover various aspects of our application's user interface, including buttons, input fields, tab bars, divider, and more. The reason for using Lit was that it being a straightforward library designed for constructing rapid, lightweight web components. Furthermore, it provides the developers a component base class that eliminates boilerplate code, offering reactive state, scoped styles, and a declarative template system. This system is characterized by its compact size, swift performance, and expressive nature.
+
+### Styles
+The design system provides a comprehensive set of styles, including typography, colors, spacing, and layout guidelines. These styles are meticulously crafted to maintain visual coherence and enhance the overall user experience. SEE WHAT CAN BE ADDED FROM FIGMA OR PROJECT INTRO PAGE.
+
+### Events
+The various events, such as focus, blur, click, etc., have the 'blr' prefix appended, allowing them to be accessed as 'blrFocus,' 'blrBlur,' 'blrClick,' and so on. Detail can be found in [Storybook documentation](https://github.com/deven-org/B01LER-Kitchen).
+
+### Guidelines
+In addition to components and styles, the design system includes detailed guidelines and best practices for design and development. These guidelines cover topics such as accessibility, responsive design, interaction patterns, and naming conventions, empowering our team to create consistent and high-quality user interfaces.
 
-Please describe briefly how you manage the APIs of your project and how to use them.
+### Figma
+Our design system is complemented by Figma, a collaborative interface design tool. Figma allows our team to create, share, and collaborate on designs in real-time, streamlining the design process and facilitating cross-functional collaboration.
```

Comment:
I'd say Figma is more than a complementation, it is a crucial part of the whole product and B01LER would not work without it. Figma is also the tool which includes the design counterpart of the components and it is the place where we intent to change the design tokens via the [tokens studio plugin](https://www.figma.com/community/plugin/843461159747178978).

Reply:
True, adjusted now.